pygfx/renderers/wgpu/_wgpurenderer.py

```diff
@@ -180,10 +213,10 @@ def pixel_ratio(self, value):
                 f"Rendered.pixel_ratio expected None or number, not {value}"
             )
 
-    def render(self, scene: WorldObject, camera: Camera):
-        """Main render method, called from the canvas."""
+    def render(self, scene: WorldObject, camera: Camera, clear=True, region=None):
+        """Main render method."""
```

Comment:
Todo: document the arguments.

Also, consider the name of what's now called the "region" argument. It is a rect (tuple of 4 numbers), and closely related to the "viewport", except that viewport is in physical coordinates and this argument is expressed in logical coordinates. Maybe "view" or "rect"?